load.php

```diff
@@ -248,6 +254,39 @@ function perflab_can_load_module( $module ) {
 	return (bool) $module();
 }
 
+/**
+ * Checks whether the given module can be loaded by a separate plugin.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module Slug of the module.
+ * @return bool Whether the module can be loaded by a separate plugin or not.
+ */
+function perflab_is_module_standalone( $module ) {
```

Comment:
While this is the name we discussed this for, maybe let's call it `perflab_is_standalone_plugin_loaded( $module )`? I think that name would be more clear than the current one here. Happy to discuss alternatives.

cc @joemcgill

Reply:
The function name suggestion look good to me.